src/templates/career-details.tsx

```diff
@@ -70,40 +68,37 @@ export const PersonioHtml = styled.div`
 
 interface CareerPageProps {
   pageContext: {
-    position: PersonioJobPosition;
-    socialCardImage: string;
-    hasTranslation: boolean;
     language: string;
     translation: string;
     i18n: {
       originalPath: string;
     };
   };
   location: Location;
+  data: {
+    syPersonioJob: SyPersonioJob;
+  };
 }
 const CareerPage: React.FC<CareerPageProps> = (props): JSX.Element => {
   const { pageContext } = props;
-  const { t } = useTranslation();
+  const position = props.data.syPersonioJob;
```

Comment:
I did only the things to keep this working. We still need to update the career details page